### DISCUSSION TITLE
♻️ 폴더 구조 리팩토링

### DISCUSSION THREAD

**Discussion by hyunjin0910** [prgrms-fe-devcourse/FEDC2_2JAVATAYO_Dongyoung#136] [Ideas]
나중에 한꺼번에 바꾸게, 생각나실때마다 적어주세요!
- [x] InputBox -> Common 폴더로 이동

**Comment by Hyevvy** [comment]
2페이지 이상에서 쓰면 Common 폴더에 있는게 좋은 거 같습니당 !! 현진님 좋은 안건 감사해용

**Reply by YuHyun-P** [reply]
그렇네요! 놓치고 있던 부분이었네요!
현진님 나이스👍👍
